api/endpoint/staking.go

```diff
@@ -6,10 +6,17 @@ import (
 	"github.com/trustwallet/blockatlas/coin"
 	"github.com/trustwallet/blockatlas/pkg/blockatlas"
 	"github.com/trustwallet/blockatlas/pkg/errors"
-	services "github.com/trustwallet/blockatlas/services/assets"
+	"github.com/trustwallet/blockatlas/pkg/servicerepo"
+	"github.com/trustwallet/blockatlas/services/assets"
 	"net/http"
 )
 
+var assetsService assets.AssetsServiceIface
```

Comment:
we do not need to store instances of services at implementation packages, they need to be stored at one place